### PR TITLE
Rename E Function Params - EntityLoadByExample()

### PR DESCRIPTION
**Pull request by shawnoden**
RE: #898

### CODE REVIEW

**Comment by shawnoden** [comment]
Note: the `matchCriteria` param is specified in the CF2018 param document, but I can't find it documented anywhere.

File: data/en/entityloadbyexample.json [hunk 1]
```diff
@@ -1,16 +1,17 @@
 {
 	"name":"entityLoadByExample",
 	"type":"function",
-	"syntax":"entityLoadByExample(sampleEntity [, unique])",
+	"syntax":"entityLoadByExample(sampleEntity [, unique, matchCriteria])",
 	"returns":"any",
 	"related":["entityLoadByPK"],
-	"description":" Loads and returns an array of objects that match the sampleentity.\nentityLoadByExample(sampleEntity,[unique])",
+	"description":"Loads and returns an array of objects that match the `sampleEntity.",
```
Comment (KamasamaK) [inline data/en/entityloadbyexample.json]: You added an opening ` but not a closing one.

Comment (shawnoden) [reply data/en/entityloadbyexample.json]: I fixed it.

**Comment by pfreitag** [comment]
Thanks

**Pull request merged**